src/helpers/AnimateButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
Not sure if this is considered a helper! or a component. Maybe the file structure needs more thinking!